### PR TITLE
[front] feat: Link to analysis page on VideoCard (#824)

### PR DESCRIPTION
**Pull request by adrausch**
**related to** #824

### CODE REVIEW

File: frontend/src/features/videos/VideoList.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 import { Typography, Box } from '@mui/material';
 import VideoCard from '../videos/VideoCard';
-import { CompareNowAction, AddToRateLaterList } from 'src/utils/action';
+import { CompareNowAction, AddToRateLaterList, AddDetailedConsultation } from 'src/utils/action';
```
Comment (lfaucon) [inline frontend/src/features/videos/VideoList.tsx:5]: There are a few checks for the code style that you can automatically fix with 

`docker exec tournesol-dev-front yarn lint:fix`

File: frontend/src/features/videos/VideoList.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ function VideoList({
   const { isLoggedIn } = useLoginState();
 
   const defaultActions = isLoggedIn
-    ? [CompareNowAction, AddToRateLaterList]
+    ? [CompareNowAction, AddToRateLaterList, AddDetailedConsultation]
```
Comment (lfaucon) [inline frontend/src/features/videos/VideoList.tsx:29]: We want the analysis page to ba a public page accessible even to users without loggin. For this you should also add the action in the line below:

```TypeScript
isLoggedIn
    ? [CompareNowAction, AddToRateLaterList, AddDetailedConsultation]
    : [AddDetailedConsultation]
```

File: frontend/src/utils/action.tsx [hunk 3]
```diff
@@ -77,3 +78,18 @@ export const RemoveFromRateLater = (asyncCallback?: () => void) => {
   };
   return RemoveFromRateLaterComponnent;
 };
+
+export const AddDetailedConsultation = ({ uid }: { uid: string }) => {
+  const { t } = useTranslation();
+  return (
+    <Tooltip title={`${t('actions.detailed')}`} placement="left">
```
Comment (lfaucon) [inline frontend/src/utils/action.tsx:85]: Screenshot 2022-04-20 at 08 03 51

The text of the tooltip is missing. You need to add it in the files `translation.json` to add text for French and English

**Review by lfaucon — CHANGES_REQUESTED** [review]
Very good work!

File: frontend/src/utils/action.tsx [hunk 4]
```diff
@@ -77,3 +78,18 @@ export const RemoveFromRateLater = (asyncCallback?: () => void) => {
   };
   return RemoveFromRateLaterComponnent;
 };
+
+export const AddDetailedConsultation = ({ uid }: { uid: string }) => {
```
[the remark this answers is not in the document]
Comment (adrausch) [reply frontend/src/utils/action.tsx:82]: ok

File: frontend/src/utils/action.tsx (hunk 3 above)
Comment (adrausch) [reply frontend/src/utils/action.tsx]: fixed by https://github.com/tournesol-app/tournesol/pull/827/commits/72fb63bd077f1c021b54efa492b27463d9323545

**Review by GresilleSiffle — APPROVED** [review]

**Pull request merged**